test_util/aws.py

```diff
@@ -48,8 +48,8 @@ def resource(self, name):
     def create_key_pair(self, key_name):
         """Returns private key of newly generated pair
         """
-        key = self.resource('ec2').KeyPair(key_name)
-        return key.key_material
+        key = self.client('ec2').create_key_pair(KeyName=key_name)
```

Comment:
The former actually doesnt work. This was untested code. The key material can only be retrieved from the client method